course/templates/course/course_base.html

```diff
@@ -79,6 +79,7 @@ <h4>{% trans "Calendar" %}</h4>
             {% language user_language %}
             <div lang={{user_language}}>
                 <div class="alert alert-default">
+                    <img src="/static/images/uls.svg" alt="" style="float: left; height: 2em; margin-right: 10px;">
```

Comment:
The style should be implemented with a class and relevant settings in the css file.